packages/cypress/src/fixture.ts

```diff
@@ -16,9 +14,11 @@ function getFixtureFile() {
 }
 
 export function initFixtureFile() {
-  debug("REPLAY_CYPRESS_UPDATE_FIXTURE: %s", process.env.REPLAY_CYPRESS_UPDATE_FIXTURE);
+  logger.info("CypressReporter:InitFixtureFile", {
```

Comment:
Won't it already be clear from passing the package name/version to the `initLogger` call that the logs are coming from the cypress package?

Usually when I format logs, I do something more like:

`InitFixtureFile:Started`

That way, it is clear which function each log comes from, and in most cases there is a verb there so it is easy to read and understand without looking at the code.

Reply:
Examples: `FunctionName:Failed` or`FunctionName:Started` or `FunctionName:WillGetSources`, etc.